<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatPanel</class>
 <widget class="QWidget" name="ChatPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>249</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="1" column="0" colspan="4">
    <widget class="QScrollArea" name="chatScroll">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="accessibleDescription">
      <string>This is the chat scroll area</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollContent">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>235</width>
        <height>329</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="buttonClear">
     <property name="toolTip">
      <string>clear the messages ...</string>
     </property>
     <property name="accessibleDescription">
      <string>Press that button to clear your message</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="buttonAutoTranslate">
     <property name="toolTip">
      <string>Auto translate</string>
     </property>
     <property name="accessibleDescription">
      <string>Turn on/off the auto translation for chat messages</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLineEdit" name="chatText">
     <property name="accessibleDescription">
      <string>Type your text to sent on the chat here</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
